ui/src/pages/SiteUsersPage/SiteUserOrgItems.js

```diff
@@ -11,25 +11,34 @@ const enhance = compose(
   }),
   withProps(({ user }) => {
     const schema = 'organisation';
-    const organisations = user
-      .get('organisations')
-      .map(org => new Map({ $oid: org }));
     const filter = new Map({
-      _id: new Map({
-        $in: organisations
-      })
+      owner: user.get('_id')
     });
 
     return { schema, filter };
   }),
   withModels
 );
 
-const SiteUserOrgItems = ({ models, user }) => {
+const SiteUserOrgItems = ({ models, user, fetchMore, hasMore }) => {
   const orgsItems = models
     .map(org => <SiteUserOrgItem key={org.get('_id').toString()} org={org} user={user} />)
     .valueSeq();
-  return <ul>{orgsItems}</ul>;
+
+  return (
+    <div>
+      <ul>{orgsItems}</ul>
+      {
+        hasMore
+          ? <button
+            className={'btn btn-default btn-sm'}
+            onClick={fetchMore}>
+            Load more
+          </button>
+          : null
+      }
+    </div>
```

Comment:
It can be replaced by a simplified construction
`{
        hasMore
          && <button
            className={'btn btn-default btn-sm'}
            onClick={fetchMore}>
            Load more
          </button>
      }`

Reply:
Personally I prefer to move conditional blocks to separate methods and naming them. This can improve code readability. Sure this should be applied to blocks of code larger than one simple tag or something

Reply:
Done!